packages/manager/apps/ips/src/utils/getLinkByServiceName.ts

```diff
@@ -0,0 +1,65 @@
+import { ClientNavigationApi } from '@ovh-ux/shell/dist/types/plugin/navigation';
+import {
+  IPRoutedServiceType,
+  getTypeByServiceName,
+} from './getTypeByServiceName';
+
+export type GetLinkByServiceNameParams = {
+  serviceName: string;
+  navigation: ClientNavigationApi;
+};
+
+export const getLinkByServiceName = ({
+  serviceName,
+  navigation,
+}: GetLinkByServiceNameParams): Promise<string> => {
+  if (!serviceName || !navigation) return Promise.resolve(null);
+  const serviceType = getTypeByServiceName({ serviceName });
```

Comment:
I don't think we get serviceType as 'HOSTING' as I dont see regEx check for housing inside getTypeByServiceName function.